docs/sts/keycloak.md

```diff
@@ -8,24 +8,22 @@ Keycloak is an open source Identity and Access Management solution aimed at mode
 - Download and start Keycloak server by following the [installation guide](https://www.keycloak.org/docs/latest/getting_started/index.html) (finish upto section 3.4)
 
 ## 2. Configure Keycloak
-
+- Go to Settings for your `demo` realm -> Enable `Implicit Flow`, then Save.
```

Comment:
`Implicit Flow Enabled` switch is available under, Clients -> Click on `account` -> Settings

Reply:
Done